package_manifest.yaml

```diff
@@ -946,6 +946,17 @@ repoclosures:
           - el8-extras
           - el8-puppet-7
           - "el8-foreman-{{ foreman_version }}-staging"
+    plugins-staging-repoclosure-el9:
+      repoclosure_target_repos:
+        el9:
+          - "el9-foreman-plugins-{{ foreman_version }}-staging"
+      repoclosure_target_dist: el9
+      repoclosure_lookaside_repos:
+        el9:
+          - el9-baseos
+          - el9-appstream
+          - el9-puppet-7
```

Comment:
I didn't add salt, as we have no config for el9-salt yet.

https://repo.saltproject.io/salt/py3/redhat/9/ exists, but for some reason we've been using https://repo.saltproject.io/py3/redhat/ for other repos (note the missing "salt" in the folder structure)

Reply:
https://docs.saltproject.io/salt/install-guide/en/latest/topics/install-by-operating-system/centos.html all uses https://repo.saltproject.io/salt/py3/redhat/ so I think we should switch over.

Reply:
https://github.com/theforeman/foreman-packaging/pull/10219